### PR TITLE
[Delta] Fixes Area Names For Deltastation & Missing Custodial Closet Area

### LINKED ISSUE #23423: Incorrect area name on DeltaStation
> Central Command Higher Dimensionsal Affairs
> 
> 
> Figments from an eldritch god are being summoned by Levi Crypt into **Library** from an unknown dimension. Disrupt the ritual at all costs!

image

That is not the Library. That is the ABANDONED LIBRARY.

### LINKED ISSUE #23469: Custodial Closet apc on delta has a space area instead of a custoidial closet area
hey look a bug

### PR DESCRIPTION
**Pull request by JohnTheDonut**
## **Fixes Area Names For Deltastation**
This renames and adds areas in the "Space Station_13_Areas.dm" and assigns said areas to Delta Station, This PR also removes MOST Var edited areas on deltastation and replaces them with proper standalone areas from the "Space Station_13_Areas.dm" file this is to allow proper and more accurate announcements for different areas on the Station. I will be following up in the near future with PR's doing this for the rest of the maps because this process is quite tedious. 

## **Custodial Closet Being Cold On Deltastation**
This PR also adds a missing area tile that is missing on top of the Deltastation Custodial Closet APC, this was previously causing the room to become cold and sets off Atmos alarms. This PR fixes that by re adding the custodial closet area. 

## **Changelog**
:cl: Tofa01
Fix: [Delta] Fixes area names for Deltastation
Fix: [Delta] Fixes custodial closet being cold all the time on Deltastation
/:cl:

### CODE REVIEW

File: code/game/area/Space_Station_13_areas.dm [hunk 1]
```diff
@@ -463,11 +519,21 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	icon_state = "medbay2"
 	music = 'sound/ambience/signal.ogg'
 
+/area/medical/medbay2/medbay_storage
+	name = "Medbay Storeage"
```
Comment (Haevacht) [inline code/game/area/Space_Station_13_areas.dm]: Storage.

Comment (Tofa01) [inline code/game/area/Space_Station_13_areas.dm]: You got me there, fixing.

**Comment by Tofa01** [comment]
Will fix map merge issue soon

**Comment by Tofa01** [comment]
Did not spell conflicts correctly in last commit, anyway i resolved conflicts using "Theirs" and redid all my changes. Should be good.

**Comment by Tofa01** [comment]
Or not... Will try and fix again.

**Comment by Haevacht** [comment]
It looks like the turf decals thing is being annoying.

**Comment by Tofa01** [comment]
Yeah it worked on local for a bit, now it is not working i will try and re do all the area name changes with a clean copy of deltastation.dm it hopefully should be ok i found a much faster way of applying the area name changes using find and replace in dream maker.

**Comment by Tofa01** [comment]
Hm, just noticed metastation decided to merge.... No clue why this is weird.

**Comment by Tofa01** [comment]
Hey! Looks like it works i fixed it, this is ready to merge whenever now i guess.

**Comment by Tofa01** [comment]
This is ready to merge whenever now i guess.

**Pull request merged**